src/charm.py

```diff
@@ -588,17 +601,17 @@ def _install_deps(self) -> None:
             env["NO_PROXY"] = self.proxies["no_proxy"]
             env["no_proxy"] = self.proxies["no_proxy"]
 
-        execute_command(["/usr/bin/apt-get", "update"])
-        # install dependencies used by repo-policy-compliance and the firewall
-        execute_command(
-            ["/usr/bin/apt-get", "install", "-qy", "gunicorn", "python3-pip", "nftables"]
-        )
+        command = [
+            "/usr/bin/pip",
+            "install",
+            "git+https://github.com/canonical/repo-policy-compliance@main",
+        ]
+
+        if upgrade_compliance is True:
+            command.insert(2, "--upgrade")
```

Comment:
Perhaps just always upgrade?

Reply:
Added a commit that is always adding the `--upgrade` option.